vnext/ReactUWP/Views/ViewManagerBase.cpp

```diff
@@ -196,6 +196,15 @@ XamlView ViewManagerBase::CreateView(int64_t tag) {
   // Set the tag if the element type supports it
   SetTag(view, tag);
 
+  // In Debug, set the element name to the tag for convienent
+  // searching within VisualStudio's Live Visual Tree pane
+#ifdef DEBUG
+  auto element = view.try_as<winrt::FrameworkElement>();
+  if (element) {
+    element.Name(std::to_wstring(tag));
```

Comment:
Might it make more sense to set the name to something like : "<ReactTag> 123"  Rather than just "123"?

Reply:
what would the space mean?  I could put a tag: prefix on the name too

Reply:
Oh, looks like my comment conflicted with markdown `<reacttag>: 123`
I had something in brackets that got replaced with a space